src/olympia/files/migrations/0006_auto_20210223_1215.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 2.2.18 on 2021-02-23 12:15
```

Comment:
Any idea how long this will take to execute on prod?

Reply:
There was about 14K files to update on prod, but this should be fairly straightforward. I don't expect more than a minute